### ISSUE TITLE
Several fields disappeared from `order.items` after upgrading v1.6.4 -> v1.6.5 🐞

### ISSUE THREAD

**Issue by magnusdr** [medusajs/medusa#2671]
GET request to `/store/orders/:id` returns different data structure after upgrading to v1.6.5. 

Here's a `git diff` after dumping the same object from `/store/orders/:id` before and after the upgrade:

```diff
$ git diff
diff --git a/medusa/order-dump.json b/medusa/order-dump.json
index 32fdc03..8ef54a1 100644
--- a/medusa/order-dump.json
+++ b/medusa/order-dump.json
@@ -109,12 +109,6 @@
           }
         ],
         "adjustments": [],
-        "subtotal": 390000,
-        "discount_total": 0,
-        "total": 487500,
-        "original_total": 487500,
-        "original_tax_total": 97500,
-        "tax_total": 97500,
         "refundable": 487500
       }
     ],
diff --git a/medusa/package.json b/medusa/package.json
index 5b97d30..93a4bf3 100644
--- a/medusa/package.json
+++ b/medusa/package.json
@@ -12,7 +12,7 @@
     "start": "medusa migrations run && medusa start --host 0.0.0.0 --port 8080"
   },
   "dependencies": {
-    "@medusajs/medusa": "1.6.4",
+    "@medusajs/medusa": "1.6.5",
     "@medusajs/medusa-cli": "^1.3.5",
     "dotenv": "^16.0.1",
     "medusa-file-spaces": "^1.3.1",
```

### Expected behavior
As this is just a patch and not a minor release of MedusaJS, I would expect no deprecations in the API.

**Comment by srindom** [comment]
Thanks for reporting this is definitely a bug. We'll make sure to expedite a fix.

Cc: @adrien2p

**Comment by adrien2p** [comment]
@magnusdr @srindom thanks for reporting it 👌 it has already been merged 2 days ago and will be available in the next release

**Comment by olivermrbl** [comment]
Resolved in #2667
